dist/receiver.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Ooyala Receiver</title>
+
+    <script type="text/javascript" src="//www.gstatic.com/cast/sdk/libs/receiver/2.0.0/cast_receiver.js"></script>
+    <script type="text/javascript" src="//player.ooyala.com/core/e48c9b51282d406f957c666149763424?plugins=bm"></script>    
+</head>
+<body>
+    <div id="splash-screen" class="fbox-container fs">
+        <img id="splash-image" src="images/logo.svg" />
+        <div id="status-cast">
+        </div>
+    </div>
+    <div id="player"></div>
+<script type="text/javascript" src="cast.js?e9d20cbdcc1904305920"></script></body>
```

Comment:
What is this "?e9d20cbdcc1904305920" used for?
This is a follow up on the question above about cast.js. How customer is going to use this? How to fix and update to ALL customer receiver without them changing their page?

Reply:
Based on our discussion, 
1. we will need to point this to a static PATH: example:
//player.ooyala.com/static/chromecast/LOCATION_OF_CORRECT_CAST.JS
This cast.js need to be proper location on candidate or sandbox or latest (production)

2. //player.ooyala.com/core/e48c9b51282d406f957c666149763424?plugins=bm
This need to be //player.ooyala.com/core/e48c9b51282d406f957c666149763424?plugins=bm,cast
Assuming that valhalla ignores non-existent plugin, we can placehold "cast" there for future update

3. We need to migrate cast.js into core (mjolnir) as part of V4 usual deployment

4. Once player make production deployment, have PBS map "cast" into proper V4 "cast" plugin.